src/bsoncxx/types/bson_value/private/value.hh

```diff
@@ -14,11 +14,12 @@
 
 #pragma once
 
+#include <bsoncxx/config/private/prelude.hh>
+
 #include <bsoncxx/private/libbson.hh>
+#include <bsoncxx/types/bson_value/value.hpp>
```

Comment:
Not sure `view.hpp` was included but not `value.hpp`. This caused issues [here](src/bsoncxx/array/element.cpp)